test/es-module/test-esm-cjs-loader-suggestions.mjs

```diff
@@ -0,0 +1,4 @@
+import '../common/index.mjs';
+
+import('../fixtures/es-modules/cjs')
+  .catch(err => console.error(err))
```

Comment:
@BridgeAR I'm having a issue here, when I run `./node test/es-modulesthis-test.mjs` does not work the CJS stuff, but when I change my cd to `test/es-modules`, and run `../../node this-test.mjs` it works, you have any suggestions or something?

Reply:
@nodejs/modules-active-members PTAL